enterprise-suite/gotests/util/util.go

```diff
@@ -124,14 +124,20 @@ func (cb *CmdBuilder) Run() (int, error) {
 		panic("unable to get command stderr pipe")
 	}
 
+	// Always capture stdout and stderr to these string builders so that
+	// we can print them out in case of errors
+	var errStdoutBuilder strings.Builder
+	var errStderrBuilder strings.Builder
+
 	// Run the command
 	if err := cmd.Start(); err != nil {
 		panic("unable to execute command")
 	} else {
-		var stdoutWriters []io.Writer
-		var stderrWriters []io.Writer
+		// These keep all stdout/stderr capture destinations
+		stdoutWriters := []io.Writer{&errStdoutBuilder}
+		stderrWriters := []io.Writer{&errStderrBuilder}
 
-		// Add string.Builder outputs
+		// Add external string.Builder outputs
```

Comment:
For the io.Copys below - are you sure those always return? In the past I ran them in goroutines, then use Wait (potentially with a timeout).